### PR TITLE
wx: fix using a default position of canvas

### LINKED ISSUE #1357: Using wxPython 4.0.0b2 problem
```
Python 3.5.3 (default, Sep  7 2017, 16:23:57) 
[GCC 6.3.0 20170406] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import vispy
>>> vispy.use('wx')
>>> print(vispy.sys_info())
Platform: Linux-4.10.0-35-generic-x86_64-with-Ubuntu-17.04-zesty
Python:   3.5.3 (default, Sep  7 2017, 16:23:57)  [GCC 6.3.0 20170406]
NumPy:    1.13.1
Backend:  wx
pyqt4:    None
pyqt5:    None
pyside:   ('PySide', '1.2.2', '4.8.7')
pyglet:   None
glfw:     None
sdl2:     sdl2 0.9.3
wx:       wxPython 4.0.0b2
egl:      None
osmesa:   OSMesa
_test:    None

App info-gathering error:
Traceback (most recent call last):
  File "/home/eldar/src/vispy/vispy/util/config.py", line 432, in sys_info
    canvas = Canvas('Test', (10, 10), show=False, app=app)
  File "/home/eldar/src/vispy/vispy/app/canvas.py", line 205, in __init__
    self.create_native()
  File "/home/eldar/src/vispy/vispy/app/canvas.py", line 222, in create_native
    self._app.backend_module.CanvasBackend(self, **self._backend_kwargs)
  File "/home/eldar/src/vispy/vispy/app/backends/_wx.py", line 227, in __init__
    p.size, style)
TypeError: Frame(): arguments did not match any overloaded call:
  overload 1: too many arguments
  overload 2: argument 4 has unexpected type 'NoneType'
```

### PR DESCRIPTION
**Pull request by eldarkg**
Fixes #1357

### CODE REVIEW

File: vispy/app/backends/_wx.py [hunk 1]
```diff
@@ -218,13 +218,18 @@ def __init__(self, *args, **kwargs):
         else:
             self._gl_context = p.context.shared.ref._gl_context
 
+        if p.position == None:
```
Comment (davidh-ssec) [inline vispy/app/backends/_wx.py]: Flake/linting is failing on travis tests. Please change this to `if p.position is None:`.

File: vispy/app/backends/_wx.py [hunk 2]
```diff
@@ -218,13 +218,18 @@ def __init__(self, *args, **kwargs):
...
+            pos = p.position
+
         if p.parent is None:
             style = (wx.MINIMIZE_BOX | wx.MAXIMIZE_BOX | wx.CLOSE_BOX |
                      wx.SYSTEM_MENU | wx.CAPTION | wx.CLIP_CHILDREN)
             style |= wx.NO_BORDER if not p.decorate else wx.RESIZE_BORDER
             style |= wx.STAY_ON_TOP if p.always_on_top else 0
-            self._frame = wx.Frame(None, wx.ID_ANY, p.title, p.position,
-                                   p.size, style)
+            self._frame = wx.Frame(None, wx.ID_ANY, p.title, pos, p.size, style)
```
Comment (davidh-ssec) [inline vispy/app/backends/_wx.py]: Flake is also complaining about this line being one character too long. Please move one or more of the parameters to the next line.

File: vispy/app/backends/_wx.py [hunk 3]
```diff
@@ -218,13 +218,18 @@ def __init__(self, *args, **kwargs):
         else:
             self._gl_context = p.context.shared.ref._gl_context
 
+        if p.position == None:
+            pos = wx.DefaultPosition
```
Comment (davidh-ssec) [inline vispy/app/backends/_wx.py]: Is this available in all modern versions of wx? Or only the newest one?

Comment (eldarkg) [reply vispy/app/backends/_wx.py]: ```grep``` show that ```wx.DefaultPosition``` used in https://github.com/wxWidgets/wxPython

File: vispy/app/backends/_wx.py (hunk 1 above)
Comment (eldarkg) [reply vispy/app/backends/_wx.py]: ok

File: vispy/app/backends/_wx.py (hunk 2 above)
Comment (eldarkg) [reply vispy/app/backends/_wx.py]: ok

File: vispy/app/backends/_wx.py (hunk 3 above)
Comment (eldarkg) [reply vispy/app/backends/_wx.py]: https://github.com/wxWidgets/wxPython/blob/8960e7ee0b6d11c5ff02f5e89665f0bdf50b7c62/src/gtk/_core.py#L2223

**Pull request merged**